nova/virt/vmwareapi/vm_util.py

```diff
@@ -1733,6 +1733,8 @@ def _process_host_stats(obj, host_reservations_map):
     # Total vcpus is the sum of all pCPUs of individual hosts
     # The overcommitment ratio is factored in by the scheduler
     threads = getattr(hardware_summary, "numCpuThreads", 0)
+    pcpus = getattr(hardware_summary, "numCpuCores", 0)
+    vcpu_ratio = (float(pcpus) / threads) if pcpus and threads else 1.0
```

Comment:
Why the `float()`? I remember that only necessary in "old times" where we didn't get a float by default if we divided 2 ints. By now it doesn't seem necessary.
```python
>>> 24/48
0.5
```

Reply:
ah right, my C brain is shining through.